hledger-lib/hledger-lib.cabal

```diff
@@ -110,6 +110,7 @@ library
       Hledger.Data.Dates
       Hledger.Data.Journal
       Hledger.Data.Ledger
+      Hledger.Data.MarketPrice
```

Comment:
PS, I think the new show function is not used yet ? Maybe you plan to use it for print [--explicit] ? Ultimately we want the ability to print any and all directives. There are some [questions to resolve](https://github.com/simonmichael/hledger/issues/475#issuecomment-274340537).